README.md

```diff
@@ -50,6 +51,7 @@ place to ask about it might be in [ipfs/apps](https://github.com/ipfs/apps) or
 * [ipfs-share](https://github.com/rameshvarun/ipfs-share) - Pastebin/Image host/File sharing application
 * [ipfs.pics](https://github.com/ipfspics/ipfspics-server) - Upload and share pics.
 * [IPFSBin](https://github.com/victorbjelkholm/ipfsbin) - Pastebin clone build.
+* [IPFSStore](https://ipfsstore.it) - File pinning for bitcoin
```

Comment:
Could you change the wording to something more like: "Pinning paid with Bitcoin" ?